BrownField/Networking/Concepts/Connectivity-Multi-region-with-vWAN/Dual-Region-with-vWAN.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Network considerations for Azure VMware Solution dual-region deployments with Secure vWAN
+description: Learn how to configure network connectivity when Azure VMware Solution private clouds are deployed in two Azure regions with Secure vWAN.
+author: jmedina
+ms.author: jmedina
+ms.date: 06/22/2023
+ms.topic: conceptual
+ms.service: caf
+ms.subservice: caf-scenario-vmware
+ms.custom: think-tank, e2e-azure-vmware
+---
+
+# Network considerations for Azure VMware Solution dual-region deployments using Secure vWAN with Routing-Intent
+
+This article describes the best practices for connectivity, traffic flows, and high availability of dual-region Azure VMWare Solution when using Azure vWAN with Routing Intent. This article will break down vWAN with Routing Intent topology from the perspective of AVS private clouds, On-premise sites, and Azure native. How to implement and configure Secure vWAN with Routing Intent is out of the scope and will not be discussed in this document.
```

Comment:
Just a note about naming of Azure services, which must be precise if this content is meant to make its way in the public doc. The official doc for Virtual WAN does not use the term "Secure vWAN". It only uses "Secured Virtual WAN Hub". Also, when submitting content for public doc, I was told that using abbreviations is not allowed - we should use "Virtual WAN" instead of "vWAN"

Reply:
Hi @fguerri, 

I updated the document and removed all abbreviations such as AVS, VNet & VWAN. I have also updated all diagrams and removed the abbreviations as well.